src/style.css

```diff
@@ -1,3 +1,124 @@
+/* Stylowanie wszystkich divów */
+/* div,
+nav {
+  border: 2px solid;
+} */
+
 body {
+  font-family: "Roboto", sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.MoyoHeader {
+  display: flex;
+  width: 100%;
+  height: 60px;
+
+  justify-content: space-between;
+  align-items: center;
+}
+
+.MoyoHeader__nav__links a {
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+}
+
+.MoyoHeader__logo {
+  margin-left: 50px;
+  border-color: blue;
+  font-size: 0;
+}
+
+.MoyoLogo__Image {
+  width: 40px;
+  height: 40px;
+  vertical-align: bottom;
+}
+
+.MoyoHeader__nav {
+  height: 100%;
+  margin-right: 50px;
+  justify-content: center;
+}
+
+.MoyoHeader__nav__links {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  height: 100%;
+  flex-grow: 1;
+  align-items: center;
+  justify-content: center;
+}
+
+.MoyoHeader__nav__links ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  height: 100%;
+  flex-grow: 1;
+  align-items: stretch;
+  justify-content: center;
+}
+
+.MoyoHeader__nav__elem {
+  margin-right: 20px;
+  flex-grow: 1;
+  display: flex;
+  align-items: center;
+  position: relative;
+}
+
+.MoyoHeader__nav__elem a {
+  height: 100%;
+  cursor: pointer;
+  align-content: center;
+  display: flex;
+  flex-wrap: wrap;
+  text-decoration: none;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+}
+
+.MoyoHeader__nav__elem a:hover {
+  text-decoration: none;
+  color: #00acdc;
+}
+
+.MoyoHeader__nav__elem img {
+  width: 20px;
+  height: 20px;
+}
+
+.MoyoHeader__nav__elem:last-child {
+  margin-right: 0;
+}
+
+[data-qa="hover"] {
+  cursor: pointer;
+  text-decoration: none;
+}
```

Comment:
you should add hover class on every element not only on one with `data-qa`, you did it above